src/layouts/Splitted.styled.jsx

```diff
@@ -47,6 +47,7 @@ export const TicketImage = styled.img`
     align-self: flex-end;
     position: fixed;
     margin-left: -50px;
+    margin-bottom: -43px;
```

Comment:
With this i can't see the image 
![imagen](https://user-images.githubusercontent.com/66505715/148619993-04ad6cd3-636d-4654-884d-7c3916c7b308.png)

Reply:
Angelica suggested this but maybe it would look better to move the text up a little ?